packages/gui/src/components/notification/Notification.tsx

```diff
@@ -1,100 +1,35 @@
-import { Flex } from '@chia-network/core';
-import { Offers as OffersIcon } from '@chia-network/icons';
-import { Trans } from '@lingui/macro';
-import { MenuItem, Typography } from '@mui/material';
 import React from 'react';
-import { useNavigate, useLocation } from 'react-router-dom';
 
-import HeightToTimestamp from '../helpers/HeightToTimestamp';
-import NFTTitle from '../nfts/NFTTitle';
-import OfferAsset from '../offers/OfferAsset';
-import NotificationPreview from './NotificationPreview';
+import type NotificationData from '../../@types/Notification';
+import NotificationType from '../../constants/NotificationType';
+import NotificationAnnouncement from './NotificationAnnouncement';
+import NotificationOffer from './NotificationOffer';
 
 export type NotificationProps = {
-  notification: {
-    id: string;
-    message: string;
-  };
+  notification: NotificationData;
   onClick?: () => void;
+  onHide?: (notificationId: string) => void;
 };
 
 export default function Notification(props: NotificationProps) {
   const {
-    notification: {
-      offer,
-      offerSummary,
-      offerData,
-      error,
-      offered,
-      requested,
-      height,
-      metadata: {
-        data: { puzzleHash },
-      },
-    },
     onClick,
+    notification,
+    notification: { type },
+    onHide,
   } = props;
-  const canCounterOffer = puzzleHash?.length > 0;
-  const navigate = useNavigate();
-  const location = useLocation();
 
-  function handleClick() {
-    onClick?.();
+  function handleHide() {
+    onHide?.(notification.id);
+  }
+
+  if (type === NotificationType.OFFER || type === NotificationType.COUNTER_OFFER) {
+    return <NotificationOffer notification={notification} onClick={onClick} onHide={handleHide} />;
+  }
 
-    if (offerSummary) {
-      navigate('/dashboard/offers/view', {
-        state: {
-          referrerPath: location.pathname,
-          offerData,
-          offerSummary,
-          imported: true,
-          canCounterOffer,
-          address: puzzleHash,
-        },
-      });
-    }
+  if (type === NotificationType.ANNOUNCEMENT) {
+    return <NotificationAnnouncement notification={notification} onClick={onClick} onHide={handleHide} />;
```

Comment:
`onHide` doesn't seem to be a supported param for `NotificationAnnouncement`

Reply:
I removed onHide (it was old code)